free-courses-pt_BR.md

```diff
@@ -59,11 +61,22 @@
 * [Introdução ao Java e Orientação a objetos](https://www.udemy.com/introducao-ao-java-e-orientacao-a-objetos/) - Helder Guimaraes Aragao (Udemy)
 
 
+### Javascript
+
+* [Fast & Furious](https://www.youtube.com/watch?v=yRr8Wo4XfYY&list=PLy5T05I_eQYOoUz2TtAqq35RLCc-xBZCe) - Codecasts (Youtube)
```

Comment:
If you look at other Youtube links in our course  lists, they point to playlists, for example

https://www.youtube.com/playlist?list=PLXFk6ROPeWoAvLMyJ_PPfu8oF0-N_NgEI

Is there a playlist link for "Fast & Furious" and the Node courses?

Reply:
Yes, there is. Thanks for your feedback!
I updated all links.